src/shell/components/RelationalFieldBase/ActiveItem/index.tsx

```diff
@@ -240,7 +240,7 @@ export const ActiveItem = memo(
 
     // Ensure item is fetched if it wasn't included in the initial 100-item fetch
     useEffect(() => {
-      if (contentItem) return;
+      if (contentItem || !relatedModelData?.ZUID || !itemZUID) return;
```

Comment:
What is this going to do to the user experience? If the related model 404s and we exit without the content item is it going to be a blank field?